tests/cli-tests/compression/basic.sh

```diff
@@ -24,6 +24,10 @@ zstd -c file       | zstd -t
 zstd --stdout file | zstd -t
 println bob | zstd | zstd -t
 
+# Test preservation in gzip mode
+(exec -a gzip zstd -c file)       | zstd -t
+(exec -a gzip zstd --stdout file) | zstd -t
```

Comment:
These tests are title "preservation in `gzip` mode".
However, both tests only check that output of `--stdout` or `-c` get properly decoded,
not explicitly that the underlying file is preserved.
I believe it should be paired with an explicit `test -f` test.

Reply:
added `test -f` to make it more explicit.